src/PdfSharper/Fonts/AFM/AFMSource.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System.Collections.Generic;
+
+namespace PdfSharper.Fonts.AFM
+{
+    public class AFMSource
+    {
+        private static readonly string COURIERBOLD = "PdfSharper.Fonts.AFM.Files.Courier-Bold.afm";
+        private static readonly string COURIEROBLIQUE = "PdfSharper.Fonts.AFM.Files.Courier-Oblique.afm";
+        private static readonly string COURIER = "PdfSharper.Fonts.AFM.Files.Courier.afm";
+        private static readonly string COURIERBOLDOBLIQUE = "PdfSharper.Fonts.AFM.Files.Courier_BoldOblique.afm";
+        private static readonly string HELVETICABOLD = "PdfSharper.Fonts.AFM.Files.Helvetica-Bold.afm";
+        private static readonly string HELVETICABOLDOBLIQUE = "PdfSharper.Fonts.AFM.Files.Helvetica-BoldOblique.afm";
+        private static readonly string HELVETICAOBLIQUE = "PdfSharper.Fonts.AFM.Files.Helvetica-Oblique.afm";
+        private static readonly string HELVETICA = "PdfSharper.Fonts.AFM.Files.Helvetica.afm";
+        private static readonly string SYMBOL = "PdfSharper.Fonts.AFM.Files.Symbol.afm";
+        private static readonly string TIMESBOLD = "PdfSharper.Fonts.AFM.Files.Times-Bold.afm";
+        private static readonly string TIMESBOLDITALIC = "PdfSharper.Fonts.AFM.Files.Times-BoldItalic.afm";
+        private static readonly string TIMESITALIC = "PdfSharper.Fonts.AFM.Files.Times-Italic.afm";
+        private static readonly string TIMESNEWROMAN = "PdfSharper.Fonts.AFM.Files.Times-New-Roman.afm";
+        private static readonly string TIMESROMAN = "PdfSharper.Fonts.AFM.Files.Times-Roman.afm";
+        private static readonly string ZAPFDINGBATS = "PdfSharper.Fonts.AFM.Files.ZapfDingbats.afm";
+
+        public static IEnumerable<string> GetAll()
+        {
+            List<string> afmList = new List<string>();
```

Comment:
Make a static readonly variable for these, this method will recreate the list every time.